src/main/resources/reference.conf

```diff
@@ -511,6 +511,41 @@ resourceTypes = {
     }
     reuseIds = true
   }
+  spend-profile = {
+    actionPatternObjects = {
+      "create" = {
+        description = "Can create profiles"
+      }
+      "update" = {
+        description = "Can update profiles"
+      }
+      "delete" = {
+        description = "Can delete profiles"
+      }
+      "list" = {
+        description = "Can read profiles"
+      }
+      "link" = {
+        description = "Can link profiles"
+      }
+      "share_policy::owner" = {
+        description = "Can grant and revoke a users' owner permission"
+      }
+    }
+    ownerRoleName = "owner"
+    roles = {
+      admin = {
+        roleActions = ["share_policy::owner"]
+      }
+      owner = {
+        roleActions = ["create", "update", "delete", "list", "link", "share_policy::owner"]
+      }
+      user = {
+        roleActions = ["list", "link", "share_policy::owner"]
```

Comment:
`share_policy::owner` does not belong in the user role

Reply:
Thank you for catching this!